app/src/main/java/de/tum/in/tumcampusapp/api/Helper.java

```diff
@@ -102,6 +103,17 @@ public Response intercept(Chain chain) throws IOException {
         };
     }
 
+    /**
+     * encodes an url
+     *
+     * @param pUrl input url
+     * @return encoded url
+     */
+    public static String encodeUrl(String pUrl) {
+        String url = pUrl.replace("/", ""); //remove slashes in queries as this breaks the url
```

Comment:
It party is only applied to the arch_id of a room.
And it is not necessary unless there is a slash in the arch_id of a room, which is not the case.

So this can be removed.